eng/pipelines/update-azure-sdk-for-net-codes.yml

```diff
@@ -1,6 +1,9 @@
 # Generate SDK codes
 
 parameters:
+  - name: NodeVersion
```

Comment:
I feel like we pass the value from outside, so that when it comes to 22.x, we don't need to change two places.

Reply:
actually we have to change 5 places now if you search `20.x` in this repo.
This is not the priority right now. We could fix it later.